frontend/src/service/Url.service.ts

```diff
@@ -179,9 +185,9 @@ export class UrlService {
   private gqlCreateURLVariable(
     captchaResponse: string,
     link: Url,
-    alias: string | null,
     isPublic: boolean = false
   ) {
+    let alias = link.alias === '' ? null : link.alias!;
```

Comment:
This part of business logic. I think it should happen outside this function. I wonder if we can remove this completely.